### ISSUE TITLE
Notices don't render colours

### ISSUE THREAD

**Issue by Polsaker** [vector-im/element-android#2690]
**Describe the bug**
When sending a `m.notice` to a channel, the message is displayed unformatted (it ignores the `format` property). This works properly in Element web

**To Reproduce**
Send a `m.notice` message that contains formatting

**Expected behavior**
The formatting is shown properly, just like in Element web

**Screenshots**
Screencap from Element Android:
![image](https://user-images.githubusercontent.com/2000719/104851148-05e1e980-58d2-11eb-9b90-9f722e00aca7.png)

Screencap from Element Web:
![image](https://user-images.githubusercontent.com/2000719/104851161-14c89c00-58d2-11eb-83f7-58ad920c8dec.png)

**Smartphone (please complete the following information):**
 - Device: Xiamomi Mi 9T
 - OS: Android 10

**Additional context**
 - App version and store 1.0.12, Google Play store
 - Homeserver: phuks.co

**Comment by xPMo** [comment]
Additionally, `m.notice` messages ignore code blocks.

This is a a total armchair guess, but it seems that `span { text; textStyle="italic"; textColor=foo }` overrides nested codeblocks and colors: https://github.com/vector-im/element-android/blob/64a37c251d76aefcdafa6aad2d19a4b3c9702659/vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt#L506

**Comment by G2G2G2G** [comment]
related https://github.com/vector-im/element-android/issues/3576

**Comment by xPMo** [comment]
If my guess is correct, my vote would be to remove the `textStyle="italic"`, since that causes more broken formatting than the `textColor`. The grey messages still distinguish `m.notice`, **and** removing italics would make formatting in line with Web/Desktop, where `m.notice`s are not italicized.

**Comment by Polsaker** [comment]
From my tests, removing `textStyle="italic"` does not fix the issue :(

**Comment by xPMo** [comment]
> From my tests, removing `textStyle="italic"` does not fix the issue :(

Well, it doesn't fix the _colors_ problem, but it should fix code blocks.

**Comment by HarHarLinks** [comment]
Colors in notices are now working for me, see examples #5438
Is this issue resolved?

**Comment by rom4nik** [comment]
> Even if this doesn't solve the underlying issue, removing italics would make formatting in line with Web/Desktop, where m.notices are not italicized.

Is there any reason as to why it's in italics on Android and regular on Web/Desktop?

**Comment by Polsaker** [comment]
> Colors in notices are now working for me, see examples #5438 Is this issue resolved?

No. It's not solved for me. I can still replicate the issue with the latest version.
